samples/sampler/stories/default/TransferList.js

```diff
@@ -16,12 +16,14 @@ export const _TransferList = (args) => (
 	<TransferList
 		allowMultipleDrag={args['allowMultipleDrag']}
 		firstList={['Item1', 'Item2', 'Item3', 'Item4', 'Item5', 'Item6', 'Item7', 'Item8']}
+		itemSize={args['itemSize']}
 		moveOnSpotlight={args['moveElementOnSpotlightDirections']}
 		secondList={['Item9', 'Item10', 'Item11', 'Item12', 'Item13', 'Item14', 'Item15', 'Item16']}
 	/>
 );
 
 boolean('allowMultipleDrag', _TransferList, Config, true);
+number('itemSize', _TransferList, Config, 201);
```

Comment:
do we need to specify 201 here if we defined it as defaultProp?

Reply:
Just as a reference for the user.